### ISSUE TITLE
Test fails in Firefox but passes in Chrome:

### ISSUE THREAD

**Issue by Bijay-Panthi** [freeCodeCamp/freeCodeCamp#15306]
Challenge [change-the-color-of-an-svg-element](https://beta.freecodecamp.com/en/challenges/data-visualization-with-d3/change-the-color-of-an-svg-element) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; rv:53.0) Gecko/20100101 Firefox/53.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<body>
  <script>
    const dataset = [12, 31, 22, 17, 25, 18, 29, 14, 9];
    
    const w = 500;
    const h = 100;
    
    const svg = d3.select("body")
                  .append("svg")
                  .attr("width", w)
                  .attr("height", h);
    
    svg.selectAll("rect")
       .data(dataset)
       .enter()
       .append("rect")
       .attr("x", (d, i) => i * 30)
       .attr("y", (d, i) => h - 3 * d)
       .attr("width", 25)
       .attr("height", (d, i) => 3 * d)
       // Add your code below this line
       .attr("fill", "navy");      
       
       // Add your code above this line
  </script>
</body>

```

**Comment by JavaRabbit** [comment]
I think I see the issue.  

You added a semicolon to the end of your line of code;
Your code was:  `.attr("fill", "navy");`

If you omit the semicolon, the test passes fine. 

![screen shot 2017-06-13 at 9 36 56 pm](https://user-images.githubusercontent.com/3488191/27115614-b6a52fc4-5080-11e7-8f3e-450428024bb4.png)

**Comment by Bijay-Panthi** [comment]
![fcc](https://user-images.githubusercontent.com/13286457/27181659-e09afa56-521b-11e7-92f0-06d04e7bfd61.png)


Has got the same issues.

**Comment by JavaRabbit** [comment]
You're missing the period.

Code needs to be `.attr("fill", "navy")`

**Comment by Enikol** [comment]
I believe, it has something to do with the browser. @JavaRabbit from your screenshot I assume you're using Chrome, right? 
For me it works in Chrome (with or without semicolon), but the same code won't work in Mozilla Firefox or Safari on my tablet. Also, in Chrome it doesn't display the rectangles for some reason, only the results of the test; in Mozilla it displays the rectangles, but the test doesn't pass:
![mozilla](https://user-images.githubusercontent.com/19998290/27235982-c16df164-52cb-11e7-84ef-80983339f9de.jpg)
![chrome](https://user-images.githubusercontent.com/19998290/27235981-c16c956c-52cb-11e7-8d31-8f213c318054.jpg)
Since @Bijay-Panthi alsos use Mozilla, @raisedadead , I think this issue might need the "Firefox" label

**Comment by rjkmurray** [comment]
It failed without a firefox label. I have issues with labels as others know

**Comment by Bijay-Panthi** [comment]
@Enikol you are right. It works in Chrome. Thanks 
@JavaRabbit I didn't miss any dot. The screenshot failed to display. Thanks for your help.

**Comment by JavaRabbit** [comment]
@Bijay-Panthi   I was wondering if that period just didn't show. Thanks for clearing that up.

**Comment by systimotic** [comment]
Thanks for looking into this! We recently fixed this (#15775), so I'm closing this issue.

Happy coding!
